### PR TITLE
add support for multiple tfms in ref generation

### PR DESCRIPTION
**Pull request by oleksandr-didyk**
Trivial improvement to allow for multiple TFMs to be specified for package generation

### CODE REVIEW

File: src/referencePackageSourceGenerator/GenerateSource/targetPackage.csproj.template [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>##Tfm##</TargetFramework>
+    <##TfmAttr##>##Tfm##</##TfmAttr##>
```
Comment (MichaelSimons) [inline src/referencePackageSourceGenerator/GenerateSource/targetPackage.csproj.template:5]: I find it a bit odd to refer to this as an attribute in the scope of an msbuild project defined in xml.  Consider naming it `TfmProperty` or `TfmPropertyName`

File: generate.sh [hunk 2]
```diff
@@ -27,10 +27,9 @@ usage() {
     echo "  Either --pkg or --pkgCsv must be specified"
     echo ""
     echo "options:"
-    echo "  --pkg <packageName,version[,tfm]>  A package and version, no spaces, separated by comma.  If the optional"
-    echo "                                       tfm is specified, it will be used in the project that restores the"
-    echo "                                       package."
-    echo "                                       Example: System.Text.Json,4.7.0"
+    echo "  --pkg <packageName,version[,tfm]>  A package and version, no spaces, separated by comma.  If optional semicolon-separated"
```
Comment (MichaelSimons) [inline generate.sh:30]: Consider renaming `tfm` to `tfms` as it betters calls out the support for multiples.

File: generate.sh [hunk 3]
```diff
@@ -27,10 +27,9 @@ usage() {
...
+    echo "                                       TFMs are specified, they will be used in the project that restores the package."
+    echo "                                       Example: System.Text.Json,4.7.0,nestandard2.0;net461"
```
Comment (MichaelSimons) [inline generate.sh:32]: Consider adding multiple examples to make it clear that the tfms are optional.
```suggestion
    echo "                                       Example: System.Collections,4.3.0"
    echo "                                       Example: System.Text.Json,4.7.0,nestandard2.0;net461"
```

File: src/referencePackageSourceGenerator/GenerateSource/targetPackage.csproj.template (hunk 1 above)
Comment (oleksandr-didyk) [reply src/referencePackageSourceGenerator/GenerateSource/targetPackage.csproj.template]: Sure thing, renamed to `TfmPropertyName`

File: generate.sh (hunk 2 above)
Comment (oleksandr-didyk) [reply generate.sh]: Good point, thanks. Renamed

File: generate.sh (hunk 3 above)
Comment (oleksandr-didyk) [reply generate.sh]: Good suggestion, thanks. Added

**Review by MichaelSimons — APPROVED** [review]

**Pull request merged**